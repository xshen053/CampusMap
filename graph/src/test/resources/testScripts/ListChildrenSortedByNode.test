# Create a graph
CreateGraph LCSBN

# Add a node
AddNode LCSBN n1
AddNode LCSBN n1
AddNode LCSBN n2
AddNode LCSBN n1

# Add an edge
AddEdge LCSBN n1 n1 eZ
AddEdge LCSBN n1 n2 eZ

# Print the nodes in the graph and the outgoing edges from n1
ListNodes LCSBN
ListChildren LCSBN n1