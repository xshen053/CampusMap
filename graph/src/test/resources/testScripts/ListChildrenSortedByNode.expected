# Create a graph
created graph LCSBN

# Add a node
added node n1 to LCSBN
added node n1 to LCSBN
added node n2 to LCSBN
added node n1 to LCSBN

# Add an edge
added edge eZ from n1 to n1 in LCSBN
added edge eZ from n1 to n2 in LCSBN

# Print the nodes in the graph and the outgoing edges from n1
LCSBN contains: n1 n2
the children of n1 in LCSBN are: n1(eZ) n2(eZ)